Bindings/Python/__init__.py

```diff
@@ -1,15 +1,19 @@
 import sys
 import os
-if (sys.platform.startswith('win')):
-      curFolder = os.path.dirname(os.path.realpath(__file__))
-      os.add_dll_directory(curFolder)
-      # in install env, add relative path from init.py to bin, in build environment, we also add "../../../../Release"
-      install_path = os.path.join(curFolder, "../../../bin")
-      if (os.path.exists(install_path)):
-          os.add_dll_directory(install_path)
-      dev_path = os.path.join(curFolder, "../../../../Release")
-      if (os.path.exists(dev_path)):
-          os.add_dll_directory(dev_path)
+
+if sys.platform.startswith('win'):
+    curFolder = os.path.dirname(os.path.realpath(__file__))
+    os.add_dll_directory(curFolder)
+    # in install env, add relative path from init.py to bin, in build environment, we also add "../../../../Release"
+    try: # pass only if DLL_PATH is updated by setup.py in local installation
+        bin_path = DLL_PATH
+        os.add_dll_directory(bin_path)
+        os.environ["PATH"] = bin_path+os.pathsep+os.environ["PATH"] # add to the beginning
+        dev_path = os.path.join(bin_path, "../Release")
```

Comment:
Python lately discouraged using PATH environment as it violates their having multiple environments with self-contained binaries/libs so I'd lean not to use PATH and instead use relative path. I understand this could be hard if trying to use executables not on the same drive or not part of the package but then this should not be built into these files which are supposed to work out of the box on any os/python version combinations.

Reply:
Adding OpenSim bin path to the `os.environ["PATH"]` was as alternative to using the system or user environment path manually. In either way, the `simbody-visualizer.exe` could be invoked properly.